test/providers/provider_level_zero.cpp

```diff
@@ -19,6 +19,89 @@
 using umf_test::test;
 using namespace umf_test;
 
+struct LevelZeroProviderInit
+    : public test,
+      public ::testing::WithParamInterface<umf_usm_memory_type_t> {};
+
+INSTANTIATE_TEST_SUITE_P(, LevelZeroProviderInit,
```

Comment:
redundant, dangling `,` at the beginning...? or is it some magic I don't understand? 😉